gridcomps/History/Sampler/MAPL_StationSamplerMod.F90

```diff
@@ -562,11 +571,13 @@ subroutine get_file_start_time(this,start_time,time_units,rc)
   end subroutine get_file_start_time
 
   ! TODO: delete and use system utilities when available
-  Subroutine count_substring (str, t, ncount)
+  Subroutine count_substring (str, t, ncount, rc)
     character (len=*), intent(in) :: str
     character (len=*), intent(in) :: t
     integer, intent(out) :: ncount
+    integer, optional, intent(out) :: rc
```

Comment:
There is no failure mode in this procedure, so maybe you did not need the change?